src/dbos.ts

```diff
@@ -1173,7 +1175,7 @@ export class DBOS {
    * @param callback - Function to run, which would call or start workflows
    * @returns - Return value from `callback`
    */
-  static async withWorkflowTimeout<R>(timeoutMS: number, callback: () => Promise<R>): Promise<R> {
+  static async withWorkflowTimeout<R>(timeoutMS: number | undefined, callback: () => Promise<R>): Promise<R> {
```

Comment:
`null` should be the semantically meaningful value to pass in here, not undefined